lib/linguist/languages.yml

```diff
@@ -5087,6 +5087,13 @@ Vue:
   tm_scope: text.html.vue
   ace_mode: html
   language_id: 391
+vvvv:
+  type: markup
+  color: "#e6e6e6"
+  extensions:
+  - ".v4p"
+  - ".vl"
```

Comment:
Yes, that first search link returns a lot of unrelated files, because other languages seem to use the `.vl` extension as well. Is that a problem?

From that second link, I've skimmed the first few pages of the search results - all of them are regular, valid vvvv / vl patches.

Reply:
> Yes, that first search link returns a lot of unrelated files, because other languages seem to use the `.vl` extension as well. Is that a problem?

Yes, with the present pull request, all `.vl` files will be recognized as vvvv. Could you update the search query provided in the original post for `.vl` to only return vvvv files (it doesn't need to be perfect, but should give us a good estimation of the number of `.vl` vvvv files)?

> From that second link, I've skimmed the first few pages of the search results - all of them are regular, valid vvvv / vl patches.

Oh, I thought there were not because they don't look much like your sample file. Maybe we need a more representative sample file? Or a few (3-4) instead of a single one?

Reply:
As I said below, I completely removed vl from this pull request to treat it as a completely separate language in the future. Which it technically isn't, but for this purpose (and training the classifier) this is the best option.